docs/user-manuals/load-aware-scheduling.md

```diff
@@ -0,0 +1,202 @@
+        plugins:
+          # enable the LoadAwareScheduling plugin
+          filter:
+            enabled:
+              - name: LoadAwareScheduling
+              ...
+          score:
+            enabled:
+              - name: LoadAwareScheduling
+                weight: 1
+              ...
+          reserve:
+            enabled:
+              - name: LoadAwareScheduling
+          ...
+        pluginConfig:
+        # configure the thresholds and weights for the plugin
+        - name: LoadAwareScheduling
+          args:
+            apiVersion: kubescheduler.config.k8s.io/v1beta2
+            kind: LoadAwareSchedulingArgs
+            # whether to filter nodes where koordlet fails to update NodeMetric
+            filterExpiredNodeMetrics: true
+            # the expiration threshold seconds when using NodeMetric
+            nodeMetricExpirationSeconds: 300
+            # weights of resources
+            resourceWeights:
+              cpu: 1
+              memory: 1
+            # thresholds (%) of resource utilization
+            usageThresholds:
+              cpu: 75
+              memory: 85
+            # the factor (%) for estimating resource usage
+            estimatedScalingFactors:
+              cpu: 80
+              memory: 70
+```
+
+The koord-scheduler takes this ConfigMap as [scheduler Configuration](https://kubernetes.io/docs/reference/scheduling/config/).
+New configurations will take effect after the koord-scheduler restarts.
+
+## Use Load Aware Scheduling
+
+1. Deploy a `stress` pod with the YAML file below.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: stress-demo
+  labels:
+    name: stress-demo
+spec:
+  containers:
+    - args:
+        - '--vm'
+        - '2'
+        - '--vm-bytes'
+        - '2G'
+        - '-c'
+        - '2'
+        - '--vm-hang'
+        - '2'
+      command:
+        - stress
+      image: polinux/stress
+      imagePullPolicy: Always
+      name: stress
+      resources:
+        limits:
+          cpu: '2'
+          memory: 4Gi
+        requests:
+          cpu: 200m
+          memory: 1Gi
+  restartPolicy: Always
+  schedulerName: koord-scheduler # use the koord-scheduler
+```
+
+```bash
+$ kubectl create -f stress-demo.yaml
+pod/stress-demo created
+```
+
+2. Watch the pod status util it becomes running.
+
+```bash
+$ kubectl get pod stress-demo -w
+NAME          READY   STATUS    RESTARTS   AGE    IP             NODE     NOMINATED NODE   READINESS GATES
+stress-demo   1/1     Running   0          20s    172.20.100.6   node-0   <none>           <none>
+```
+
+The pod is scheduled on `node-0`.
+
+3. Check the load of each node.
+
+```bash
+$ kubectl top node
+NAME     CPU(cores)   CPU%   MEMORY(bytes)   MEMORY%
+node-0   6450m        19%    25091Mi         19%
+node-2   2687m        8%     17535Mi         13%
+node-1   3280m        10%    9631Mi          7%
+```
+
+In above order, `node-0` has the highest load, while `node-1` has the lowest load.
```

Comment:
node-2 is the lowest cpu... which is easy to misunderstand, set both cpu and memory are the lowest

Reply:
done